brooklyn-server/software/base/src/main/java/org/apache/brooklyn/entity/software/base/lifecycle/MachineLifecycleEffectorTasks.java

```diff
@@ -203,6 +213,12 @@ public Void call(ConfigBag parameters) {
         public Void call(ConfigBag parameters) {
             Collection<? extends Location> locations = null;
 
+            Boolean isSkipInstallation = parameters.get(StartSoftwareParameters.SKIP_INSTALLATION);
+            if (isSkipInstallation != null && isSkipInstallation) {
+                entity().config().set(BrooklynConfigKeys.SKIP_ENTITY_INSTALLATION, isSkipInstallation);
+                entity().config().set(BrooklynConfigKeys.SKIP_ENTITY_CUSTOMIZATION, isSkipInstallation);
```

Comment:
It's dangerous to set the parameter as config value as it would be (unexpectedly) used for future calls as well. I'd suggest using curl to change the config value if needed and then do a regular `start`.